src/java.desktop/share/classes/javax/swing/text/html/HTMLEditorKit.java

```diff
@@ -1182,6 +1181,15 @@ public void handleEndOfLineString(String eol) {
      * A factory to build views for HTML.  The following
      * table describes what this factory will build by
      * default.
+     * <p>
+     * Parsed tags that lack full support are handled differently by the editor.
+     * When the container using HTML tags is editable, all non-supported tags
+     * will be displayed as editable text fields in place of the tags. The contents
+     * within the tags will be handled by the editor as regular text.
+     * If the container is not editable and a tag is recognized but not supported,
+     * such as script tags, the tag and its contents will be hidden.
```

Comment:
another nl here I think. This is almost like a list, but the main point is it is easier to read the conditions when there's a line break. Remember your formatting doesn't get carried over into the published javadoc